### ISSUE TITLE
Mapping not completing due to basisofRecord (again)

### ISSUE THREAD

**Issue by ibdj** [gbif/portal-feedback#4317]
Hi GBIF, 

I am trying to upload an occurrence data set, but when mapping to the DWC the BasisofRecord seem to 'block'. I have tried including it in the file, and also deleting it from the file and using the predefined vocabulary instead, but it simply won't save when I do that. None of those worked. Any tips would be helpful. Thank you in advance.

I have had a similar problem before (https://github.com/gbif/portal-feedback/issues/3983), where the problem was a duplicate occurenceID, but I have checked for that this time, and there shouldn't be any. 
The dataset is here: https://danbif.au.dk/ipt/manage/resource.do?r=salt_marshes_1972
Thank you very much in advance.

**Comment by ManonGros** [comment]
Hi @IBDJ thanks for reporting the issue!
We can't access the dataset if it isn't public. Would it be possible to make it public or send us the archive? Thanks!

**Comment by ibdj** [comment]
Would I be able to make it public if the occurences haven’t loaded bc of the mapping not conpleting? 
I’ll try, or I can definitely send it like last time. Thank you for getting back @ManonGros

**Comment by ManonGros** [comment]
Maybe just send it like last time @IBDJ, it will be easier

**Comment by ibdj** [comment]
Yes, I just tried and the datasat can't be published when it hasn't mapped. The dataset is here: https://naturinstitut-my.sharepoint.com/:u:/g/personal/idja_natur_gl/EfMMge6MdgxCgsQpaaGiZqwBznOy8EHDI8yptdwJu7_5uQ?e=sbdMS5

**Comment by ManonGros** [comment]
Thanks @IBDJ, I published your data on our test system via our test IPT: https://www.gbif-uat.org/dataset/efbdfa35-4fca-4f2f-ba5f-3a54b3de0f85 I didn't encounter the issue you mentioned.
could you send us a screenshot of the issue in the IPT?

**Comment by ibdj** [comment]
That is quite strange. 
What I am encountering is that the basisOfRecord is in red, and saving or adding a translation doesn't help. I can't choose from the drop-down many either: 
![image](https://user-images.githubusercontent.com/33222877/194611934-dfb80f3d-cb1d-42cb-acc7-7f8cac18c862.png)
@ManonGros

**Comment by ManonGros** [comment]
@IBDJ could you ask your IPT administrator to give me temporary access to the IPT and resource? Let's use the helpdesk@gbif.org email address to communicate credentials.

**Comment by ibdj** [comment]
Sure, I can try that. We could also schedule a meeting, and I can share my screen?
